app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -249,7 +249,7 @@ class MainViewModel @Inject constructor(
                 }
             }
 
-            val mailbox = currentMailbox.value!!
+            val mailbox = currentMailbox.value ?: return@launch
```

Comment:
We should talk about that in meeting to know what to do for know, because it's probably not good to let the user on a "bugged" screen without mailbox